ES/ESdmb003.xml

```diff
@@ -44,51 +44,97 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                        GrohHymn pp. 336-47.</note>
+                     <locus from="6ra" to="106va"/>
+                     <title type="complete" ref="LIT2384Taamme"/>
+                     <textLang mainLang="gez"/>
+                                          
+                     <msItem xml:id="ms_i1.1">
+                        <locus from="6ra" to="10rb"/>
+                        <title>Maṣḥafa śǝrʿāt and hymns</title>
+                        <textLang mainLang="gez"/>
+                        
+                        <msItem xml:id="ms_i1.1.1">
+                           <locus from="6ra" to="9rb"/>
+                           <title type="complete" ref="LIT1968Mashaf">Maṣḥafa śǝrʿāt, Introductory rite of al-Muʿallaqah</title>
+                           <textLang mainLang="gez"/>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i1.1.2">
+                           <locus from="9rb" to="9va"/>
+                           <title type="complete" ref="LIT3093RepCh373">ʾO-ʾǝgzǝʾtǝya qǝddǝst dǝngǝl Māryām Mārihām ʾǝmmu la-ʾIyasus kahāli ...
+                              "O my Lady, Holy Virgin Mary, Mother of Jesus the Almighty ..."</title>
+                           <textLang mainLang="gez"/>
+                           <incipit xml:lang="gez">ኦእግዝእትየ፡ ቅድስት፡ ድንግል፡ ማርያም፡ ማሪሃም፡ እሙ፡ ለኢየሱስ፡ ከሃሊ፡</incipit>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i1.1.3">
+                           <locus from="9va" to="10rb"/>
+                           <title type="complete" ref="LIT5443Salamlaki2">Salām laki ʾo-ʾǝgzǝʾtǝya Māryām dǝngǝl ʾǝmma madḫǝn wa-ʾǝmma bǝrhān ...
+                              "Salutation to you, o my Lady, Virgin Mary, Mother of the Saviour and Mother of the Light ..."</title>
+                           <textLang mainLang="gez"/>
+                           <incipit xml:lang="gez">ሰላም፡ ለኪ፡ ኦእግዝእትየ፡ ማርያም፡ ድንግል፡ እመ፡ መድኅን፡ ወእመ፡ ብርሃን፡</incipit>
+                        </msItem>
+                        
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i1.2">
+                        <locus from="10rb" to="11vb"/>
+                        <title ref="LIT3584Intro">Introductory exhortation</title>
+                        <textLang mainLang="gez"/>
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i1.3">
+                        <locus from="9ra" to="10ra"/>
+                        <title type="complete" ref="LIT3058RepCh338"/>
+                        <textLang mainLang="gez"/>
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i1.4">
+                        <locus from="12va" to="106va"/>
+                        <title type="complete" ref="LIT3585Taamme">103 Miracles of Mary</title>
```

Comment:
LIT5673AkkonuColl (this would also solve the problem of having two msItems for the miracles and Akko-nu besi hymns

Reply:
but here for 1.4 or above for 1? the record LIT5673AkkonuColl  seems also to have the intro parts

Reply:
I would add it as an msItem in ms_i1. Then the introductory textparts can still have their individual IDs within LIT5673AkkonuColl , and the miracles not belonging to this collection follow afterwards

Reply:
done (instead of LIT2384Taamme)

Reply:
please check also: LIT5673AkkonuColl says that the first stanza is followed by LIT5052SobaAshafa

Here it is followed by the very similar but seemingly different (?) LIT5049SobaA